### ISSUE TITLE
suggest

### ISSUE THREAD

**Issue by lgwebdream** [fighterN1/ThumbPart2#1]
1.Homework as much as possible to write their own, to learn the things applied to become their own knowledge.

**Comment by fighterN1** [comment]
You are right.I'm working on it and putting it into practice. Now, trust me, I will try。
